crypto3/crypto3.nix

```diff
@@ -33,6 +31,7 @@ in stdenv.mkDerivation {
       (if enableDebug then "-DCMAKE_BUILD_TYPE=Debug" else "-DCMAKE_BUILD_TYPE=Release")
       (if enableDebug then "-DCMAKE_CXX_FLAGS=-ggdb" else "")
```

Comment:
let's keep CXX flags in CMake (like in zkEVMFramework)

Reply:
Done